main/fzf/template.py

```diff
@@ -17,9 +17,21 @@ def post_install(self):
     self.install_man("man/man1/fzf.1")
     self.install_file("plugin/fzf.vim", "usr/share/vim/vimfiles/plugin")
     self.install_file("plugin/fzf.vim", "usr/share/nvim/runtime/plugin")
+    self.install_bin("bin/fzf-tmux")
+    self.install_man("man/man1/fzf-tmux.1")
 
     with self.pushd("shell"):
         self.install_completion("completion.bash", "bash")
         self.install_completion("completion.zsh", "zsh")
+
         for ext in ["bash", "fish", "zsh"]:
             self.install_file(f"key-bindings.{ext}", "usr/share/fzf")
+
+
+@subpackage("fzf-tmux")
+def _(self):
+    self.subdesc = "tmux integration script"
+    self.depends = [self.parent, "bash", "tmux"]
+    self.install_if = [self.parent, "bash", "tmux"]
+
+    return ["usr/bin/fzf-tmux", "usr/share/man/man1/fzf-tmux.1"]
```

Comment:
can just use `cmd:fzf-tmux` here (should be good after this, unless q66 has something to say perhaps)